src/components/elements/section.js

```diff
@@ -2,7 +2,7 @@ import cx from 'classnames'
 import factory from './factory'
 
 export default ({ property, topLevel }) => (
-  <div className={cx({'nested': !topLevel}, 'ct-section')}>
+  <div className={cx({'ct-nested': !topLevel}, 'ct-section')}>
```

Comment:
Did this break https://github.com/cignium/hypermedia-client/commit/29c4ae1ed899c117d395adad185d18b33da42345? Can't see it in this file for some reason.